crypto_kem/kyber768/clean/indcpa.c

```diff
@@ -133,7 +133,7 @@ void PQCLEAN_KYBER768_gen_matrix(polyvec *a, const unsigned char *seed,
     uint16_t val;
     unsigned int nblocks;
     const unsigned int maxnblocks = 4;
-    uint8_t buf[SHAKE128_RATE * maxnblocks];
+    uint8_t buf[SHAKE128_RATE * /* maxnblocks = */ 4];
```

Comment:
why this change?

Reply:
Microsoft compiler fails, complaining that arrays need to be initialized using constant size values.  (Yes, I'm aware that maxnblocks was declared const, but Microsoft's compiler is apparently not aware.)